docs/Data-Harmonization.md

```diff
@@ -146,6 +146,7 @@ The taxonomy can be automatically added by the taxonomy expert bot based on the
 |ddos|Availability|This IOC refers to various parts of the DDOS infrastructure.|
 |dropzone|Information Content Security|This IOC refers to place where the compromized machines store the stolen user data.|
 |phishing|Fraud|This IOC most often refers to a URL, which is phishing for user credentials.|
+|proxy|Suspicious|This refers to the use of proxies from inside your network.|
```

Comment:
According to eCSIRT, `Suspicious` needs to be replaced by `Other`. @aaronkaplan double-check please.